AssertionError while using ets from wrapper.

## Description
ets from statforecast model producing error.

## To Reproduce
(Please provide minimal example of code snippet that reproduces the error. For existing examples, please provide link.)

```python
from gluonts.ext.statsforecast import ETSPredictor
from gluonts.dataset.repository.datasets import get_dataset
from gluonts.evaluation.backtest import backtest_metrics
dataset = get_dataset("pedestrian_counts")
predictor = ETSPredictor(prediction_length=dataset.metadata.prediction_length)
agg_metrics, agg_perite = backtest_metrics(dataset.test, predictor)

```

## Error message or code output
(Paste the complete error message, including stack trace, or the undesired output that the above snippet produces.)

```
RemoteTraceback                           Traceback (most recent call last)
RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.7/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/home/bhaskar/Desktop/gluonts/src/gluonts/evaluation/_base.py", line 56, in worker_function
    return evaluator.get_metrics_per_ts(ts, forecast)
  File "/home/bhaskar/Desktop/gluonts/src/gluonts/evaluation/_base.py", line 343, in get_metrics_per_ts
    median_fcst = forecast.quantile(0.5)
  File "/home/bhaskar/Desktop/gluonts/src/gluonts/model/forecast.py", line 726, in quantile
    quantiles, quantile_predictions
  File "/home/bhaskar/Desktop/gluonts/src/gluonts/model/forecast.py", line 52, in __init__
    ), "Need at least two points for linear interpolation."
AssertionError: Need at least two points for linear interpolation.
"""
```

## Environment
- Operating system:Ubuntu 18.04
- Python version:3.7
- GluonTS version:dev
- MXNet version:

(Add as much information about your environment as possible, e.g. dependencies versions.)

--- a/src/gluonts/model/forecast.py
+++ b/src/gluonts/model/forecast.py
@@ -718,7 +718,7 @@ class QuantileForecast(Forecast):
 
         inference_quantile = Quantile.parse(inference_quantile).value
 
-        if len(quantiles) == 1 or inference_quantile in quantiles:
+        if len(quantiles) < 2 or inference_quantile in quantiles:
             q_str = Quantile.parse(inference_quantile).name
             return self._forecast_dict.get(q_str, self._nan_out)
 

--- a/test/model/test_forecast.py
+++ b/test/model/test_forecast.py
@@ -53,7 +53,7 @@ MULTIVARIATE_FORECASTS = {
 
 
 @pytest.mark.parametrize("name", FORECASTS.keys())
-def test_Forecast(name):
+def test_forecast(name):
     forecast = FORECASTS[name]
 
     def percentile(value):
@@ -73,6 +73,23 @@ def test_Forecast(name):
     assert forecast.index[0] == START_DATE
 
     forecast.plot()
+
+
+def test_mean_only_forecast():
+    forecast = QuantileForecast(
+        forecast_arrays=np.ones(shape=(1, 12)),
+        start_date=pd.Period("2022-03-04 00", freq="H"),
+        forecast_keys=["mean"],
+    )
+
+    assert forecast.prediction_length == 12
+    assert len(forecast.index) == 12
+    assert forecast.index[0] == pd.Period("2022-03-04 00", freq="H")
+
+    for level in [0.1, 0.5, 0.7]:
+        assert np.isnan(forecast.quantile(level)).all()
+    
+    assert np.equal(forecast.mean, 1).all()
 
 
 @pytest.mark.parametrize(

--- a/test/model/test_forecast.py
+++ b/test/model/test_forecast.py
@@ -88,7 +88,7 @@ def test_mean_only_forecast():
 
     for level in [0.1, 0.5, 0.7]:
         assert np.isnan(forecast.quantile(level)).all()
-    
+
     assert np.equal(forecast.mean, 1).all()
 
 

